nvflare/app_opt/lr/np_persistor.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import numpy as np
+
+from nvflare.app_common.np.np_model_persistor import NPModelPersistor
+
+
+class NewtonRaphsonModelPersistor(NPModelPersistor):
```

Comment:
this class has nothing to do with Newton-Raphson method though.

It should be called either LinearModelPersistor or LogisticRegressionModelPersistor